test/evmc/mocked_host.hpp

```diff
@@ -18,27 +18,14 @@ using bytes = std::basic_string<uint8_t>;
 /// Extended value (by dirty flag) for account storage.
 struct storage_value
 {
-    /// The storage value.
-    bytes32 value;
+    /// The current storage value.
+    bytes32 current;
 
-    /// True means this value has been modified already by the current transaction.
-    bool dirty{false};
+    /// The original storage value.
+    bytes32 original;
 
     /// Is the storage key cold or warm.
-    evmc_access_status access_status{EVMC_ACCESS_COLD};
-
-    /// Default constructor.
-    storage_value() noexcept = default;
-
-    /// Constructor.
-    storage_value(const bytes32& _value, bool _dirty = false) noexcept  // NOLINT
-      : value{_value}, dirty{_dirty}
-    {}
-
-    /// Constructor with initial access status.
-    storage_value(const bytes32& _value, evmc_access_status _access_status) noexcept
```

Comment:
These two constructors were not used, so probably fine to remove now (will be added back properly by the EVMC upgrade).

Reply:
Yes, I removed them instead of modifying. The EVMC upgrade will bring them back in a different form.